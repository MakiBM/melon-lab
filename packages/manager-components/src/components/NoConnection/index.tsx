import React, { StatelessComponent } from 'react';

import styles from './styles.css';

const NoConnection: StatelessComponent = () => {
  return (
    <div className="no-connection">
      <style jsx>{styles}</style>
      <h3>Welcome to the future of investment funds</h3>
      <p>
        It seems like you are not connected to the ethereum network. Check your
        internet connection. If you are running your own node, make you are
        synced and have enough peers connected.
      </p>
    </div>
  );
};

export default NoConnection;
