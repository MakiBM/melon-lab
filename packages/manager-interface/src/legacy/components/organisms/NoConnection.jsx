import React from 'react';
import { Card } from 'semantic-ui-react';

const NoConnection = () => (
  <div>
    <Card centered>
      <Card.Content>
        <Card.Header>Welcome to the future of investment funds</Card.Header>
        <br />
        <br />
        <p>
          {' '}
          <strong>
            It seems like you are not connected to the ethereum network. Check
            your internet connection. If you are running your own node, make you
            are synced and have enough peers connected.
          </strong>
        </p>
        <br />
      </Card.Content>
    </Card>
  </div>
);

export default NoConnection;
