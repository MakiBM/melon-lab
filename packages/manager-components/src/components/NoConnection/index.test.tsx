import React from 'react';
import NoConnection from './index';

describe('NoConnection', () => {
  const defaultElement = <NoConnection />;

  it('should render correctly', () => {
    const wrapper = shallow(defaultElement);
    expect(wrapper).toMatchSnapshot();
  });
});
