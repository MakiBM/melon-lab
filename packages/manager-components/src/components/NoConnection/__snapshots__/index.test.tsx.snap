// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoConnection should render correctly 1`] = `
<div
  className="jsx-__scopedHash no-connection"
>
  <JSXStyle
    css="__scoped"
    styleId="__scopedHash"
  />
  <h3
    className="jsx-__scopedHash"
  >
    Welcome to the future of investment funds
  </h3>
  <p
    className="jsx-__scopedHash"
  >
    It seems like you are not connected to the ethereum network. Check your internet connection. If you are running your own node, make you are synced and have enough peers connected.
  </p>
</div>
`;
